runtime/near-vm-runner/src/tests/rs_contract.rs

```diff
@@ -13,18 +13,13 @@ use crate::tests::{
     PREDECESSOR_ACCOUNT_ID, SIGNER_ACCOUNT_ID, SIGNER_ACCOUNT_PK,
 };
 
-#[cfg(feature = "protocol_feature_alt_bn128")]
-lazy_static_include::lazy_static_include_bytes! {
-    TEST_CONTRACT => "tests/res/nightly_test_contract_rs.wasm",
-}
-
-#[cfg(not(feature = "protocol_feature_alt_bn128"))]
-lazy_static_include::lazy_static_include_bytes! {
-    TEST_CONTRACT => "tests/res/test_contract_rs.wasm",
-}
-
 fn test_contract() -> ContractCode {
-    ContractCode::new(TEST_CONTRACT.to_vec(), None)
+    let code = if cfg!(feature = "protocol_feature_alt_bn128") {
```

Comment:
This seems off to me. It should be based on whether `nightly_protocol` is enabled.

Reply:
Probably! I've tried to keep pre-existing semantics in this PR, I can change this and (that)[https://github.com/near/nearcore/pull/4160/files#r600402157] in a follow up.

Reply:
Took a look here, I am not sure how to fix it. This crate doesn't have `nightly_protocol` feature: 

```toml
[features]
# all vms enabled for tests, but only one default vm, specified by runtime crate
default = ["wasmer0_vm", "wasmtime_vm", "wasmer1_vm"]
wasmer0_vm = [ "wasmer-runtime" ]
wasmtime_vm = [ "wasmtime", "anyhow"]
wasmer1_vm = [ "wasmer", "wasmer-types", "wasmer-compiler-singlepass",  "wasmer-compiler-cranelift", "wasmer-engine-native", ]
lightbeam = ["wasmtime/lightbeam"]
no_cpu_compatibility_checks = []
protocol_feature_evm = ["near-primitives/protocol_feature_evm", "near-evm-runner/protocol_feature_evm"]

no_cache = []

protocol_feature_alt_bn128 = [
    "near-vm-logic/protocol_feature_alt_bn128",
    "near-primitives/protocol_feature_alt_bn128",
    "near-vm-errors/protocol_feature_alt_bn128"
]
```

At this point, I don't know the purpose of `nightly_protocol` and `nightly_protocol_features` cargo features and why they are set up that way, so I can't immediately fix that. Learning more about our feature setup is on my todo list, so I might come back to this later.

Reply:
@matklad nightly_protocol is supposed to be a marker feature https://github.com/near/NEPs/issues/126#issuecomment-712130812